execution/executor/src/block_processor.rs

```diff
@@ -85,7 +85,7 @@ where
         committed_timestamp_usecs: u64,
         previous_state_root_hash: HashValue,
         previous_frozen_subtrees_in_accumulator: Vec<HashValue>,
-        previous_num_leaves_in_accumulator: u64,
```

Comment:
I think we used u64 to ensure it works on 32bit platforms?

Reply:
That's a fair point and a good reason for a distinct type.

Reply:
I'm pretty sure our stuff won't work on 32-bit platforms and something will break. At one point I wanted to put `static_assert(sizeof(usize) == 8)` somethere.

Reply:
Yes I'm thinking shall we define a type or assert somewhere that usize is 64 bits long?

Reply:
I was thinking exactly the same thing.  It is fine for us to depend on this behavior as long as we assert for it.

Reply:
Just curious: if we're committed to living in an u64 world anyway, why use usize?

Reply:
Lots of code in storage uses computation based on vec.len(), and it is tedious do convert back to u64 all over the place.  Maybe a type alias (to usize) would be cleaner, but then it needs to be imported everywhere or use some_horrible_crate::syntax; neither is as simple as just using usize.